src/components/UpdateArticleVotes.jsx

```diff
@@ -0,0 +1,27 @@
+import axios from "axios"
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+
+
+function UpdateArticleVotes (){
+    const article = useParams()
+    const articleChoice = article.article_id
+    let votes = 0
+    let [newVotes, setNewVotes] = useState(votes)
+    useEffect(()=>{
```

Comment:
It would be nice to have some state that disables the button while the request is being processed